wiki/Medals/Legacy_Beatmap_Packs/id.md

```diff
@@ -0,0 +1,406 @@
+# Beatmap pack lama
```

Comment:
`Paket-paket beatmap lama`

not sure what to translate `legacy` as but yeah i suppose this should be okay as `Paket beatmap warisan` or `Paket beatmap pusaka` doesn't really make sense